api/helpers/models/audit.js

```diff
@@ -13,7 +13,7 @@ module.exports = require('../models')('Audit', {
     timestamp: { type: Date, default: Date.now() },
 
     // Permissions
-    write: [{ type: String, trim: true, default: '["project-system-admin"]' }],
-    read: [{ type: String, trim: true, default: '["project-system-admin"]' }],
-    delete: [{ type: String, trim: true, default: '["project-system-admin"]' }]
+    write: [{ type: String, trim: true, default: '["sysadmin"]' }],
```

Comment:
again having a place that you call what role, as oppose to hardcoding it might be better.
* it prevents typos
* if it works one place it should work every were
* if you need to update it, (probably you won't) it is one place